src/shared/dropdown-wrapper/element-with-dropdown.tsx

```diff
@@ -176,32 +245,14 @@ export const ElementWithDropdown = ({
                     <FloatingFocusManager
                         context={context}
                         modal={false}
-                        initialFocus={dropdownRef}
+                        initialFocus={-1}
```

Comment:
the wrapper component was removed

but this disables automatic focus, components will handle their own dropdown focus behaviour anyway

didn't observe any change in behaviour